src/js/shield_canvas_draw.js

```diff
@@ -16,22 +16,27 @@ const minGenericShieldWidth = 20 * PXR;
 const maxGenericShieldWidth = 34 * PXR;
 const genericShieldFontSize = 18 * PXR;
 
-function ellipse(params, ref) {
+export function computeWidth(rectWidth, ref) {
```

Comment:
It looks like this function is conflating all shields in how they calculate width. Before/after:

![old-pentagon](https://user-images.githubusercontent.com/1732117/215277606-9cf11676-07b5-4179-a5e4-d91e12f4a400.png) ![old-trapezoid](https://user-images.githubusercontent.com/1732117/215277607-8a2485a8-2a0b-40ae-9dd8-a250130f7894.png) ![old-hexagon](https://user-images.githubusercontent.com/1732117/215277700-62c70cf1-5f43-4f7b-b21f-15edd93f58bf.png)
![new-pentagon](https://user-images.githubusercontent.com/1732117/215277610-752f81d5-78d6-40da-b33b-a267ee3f4c35.png) ![new-trapezoid](https://user-images.githubusercontent.com/1732117/215277609-2d8ea80c-cc31-4ce1-9322-5be9f4412f59.png) ![new-hexagon](https://user-images.githubusercontent.com/1732117/215277699-d90586ba-0a01-46b4-9986-4c6d0e807746.png)

Reply:
Good catching. I fixed this in 1388532, and switching between two tabs (remote and branch) seem to show no pixel variations.